src/main/service/dataManipulationService.ts

```diff
@@ -542,12 +542,12 @@ export class DataManipulationService {
       sitting['durationAsHours'] = durationAsHours;
       sitting['durationAsMinutes'] = durationAsMinutes;
       sitting['durationAsDays'] = durationAsDays;
-
-      sitting['time'] = moment.utc(sitting['sittingStart']).tz(this.timeZone).format('HH:mm');
       const min = moment(sitting['sittingStart'], 'HH:mm').minutes();
       if (min === 0) {
```

Comment:
It looks like startTime is identical to time, could this be used instead?

Reply:
Fixed as discussed on slack